plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/TestKafkaWithConfluentLatestConnectorTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.kafka.schema.confluent.KafkaWithConfluentSchemaRegistryQueryRunner;
+import io.trino.testing.BaseConnectorTest;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import io.trino.testing.kafka.TestingKafka;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static io.trino.testing.MaterializedResult.resultBuilder;
+import static io.trino.tpch.TpchTable.CUSTOMER;
+import static io.trino.tpch.TpchTable.LINE_ITEM;
+import static io.trino.tpch.TpchTable.NATION;
+import static io.trino.tpch.TpchTable.ORDERS;
+import static io.trino.tpch.TpchTable.PART;
+import static io.trino.tpch.TpchTable.PART_SUPPLIER;
+import static io.trino.tpch.TpchTable.REGION;
+import static io.trino.tpch.TpchTable.SUPPLIER;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestKafkaWithConfluentLatestConnectorTest
+        extends BaseConnectorTest
+{
+    private TestingKafka testingKafka;
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        testingKafka = closeAfterClass(TestingKafka.createWithSchemaRegistry());
+        return KafkaWithConfluentSchemaRegistryQueryRunner.builder(testingKafka)
+                .setTables(ImmutableList.of(ORDERS, LINE_ITEM, NATION, REGION, PART, PART_SUPPLIER, SUPPLIER, CUSTOMER))
+                .setExtraKafkaProperties(ImmutableMap.<String, String>builder()
+                        .put("kafka.confluent-subjects-cache-refresh-interval", "1ms")
```

Comment:
can this test be executed in parallel?

Reply:
I will try. If I see failures in ci then I may have to break it up though. Will update. lmk if that's what you meant.